eng/pipelines/templates/stages/publish.yml

```diff
@@ -198,98 +232,98 @@ stages:
         - template: /eng/pipelines/templates/variables/globals.yml
 
       jobs:
-        - deployment: Upload_Version_Txt
+        - deployment: ReleaseGate
```

Comment:
Why is this needed given you already have other deployment jobs?

Reply:
The delay between a release publishing and Choco/WinGet having the latest version can be "long". Adding an approval allows azd to launch the version.txt update once the release is live everywhere.